requirements.txt

```diff
@@ -15,6 +15,8 @@ django-filter==1.0.4
 django-rest-swagger==2.1.2
 djangorestframework==3.7.3
 docutils==0.14
+ecdsa==0.13
```

Comment:
Does the `ecdsa` package need to be included here, or is it already a dependency of `pycrypto` or `python-jose`?

Reply:
I rebuilt the venv to make sure they're all correct, installed as part of jose.

Reply:
should drop that from our requirements.txt then, and let jose handle it?

Reply:
It all needs to be pinned to prevent version errors.  This is simply the output of `pip freeze > requirements.txt` which is the standard way to lock it down.

Reply:
Yep, but then you end up with your dependencies' dependencies in requirements.txt. I've always included only the top-level packages, and let the packages take care of their own dependencies. Am I right @andyhd ? Or does this just not matter very much 😬

Reply:
I think you should simply rely on `pip freeze` and not craft it yourself.  

Crafting it yourself also often leaves random packages added to the list and no one knows what's been added, then have to delete the venv and build again to check etc etc.

Reply:
I usually do pip freeze. It might be worth switching to [pipenv](https://github.com/pypa/pipenv) for this sort of dependency locking.